### DISCUSSION TITLE
Airspy R2 Issues with SDRTrunk

### DISCUSSION THREAD

**Discussion by radiod42** [DSheirer/sdrtrunk#2012] [Q&A] [answered]
Looking for assistance with running an Airspy R2 using SDRTrunk v0.6.0.  Every time I try to run it, the tuner errors out after about 5-10 minutes.  Doesn't matter if I am using 10 MHz or 2.5 MHz sample rate.  It is on a standalone USB 3.0.  I started with 3 other RTL-SDR Blog V3s running so thought that might be the issue, but it persists even when they are unplugged.  

Recurring error below and included in the attached app log file is what seems to be causing the tuner to error then the program to freeze and crash/close.

ERROR i.g.d.s.t.u.USBTunerController - Attempt to resubmit previous error USB transfer buffer failed with status [LIBUSB_ERROR_BUSY] - this may be temporary and has happened [19350] times so far.  Transfer buffer holding queue (error/total) [4/8]  [198MB/682MB 29%]

Any assistance would be appreciated. 

Windows 10
Intel i7-11700F
16GB RAM
Only 27% CPU and 50% Memory usage

[error_sdrtrunk_app.log](https://github.com/user-attachments/files/17359772/error_sdrtrunk_app.log)

**Comment by DSheirer** [comment, marked as answer]
This issue is fixed in the latest nightly release.
